snap/snapcraft.yaml

```diff
@@ -0,0 +1,121 @@
+name: boinc
+base: core22
+version: "7.23.0"
+summary: BOINC lets you help cutting-edge science research using your computer.
+description: |
+  BOINC lets you help cutting-edge science research using your computer. The BOINC app, running on your computer, downloads scientific computing jobs and runs them invisibly in the background. It's easy and safe.
+  About 30 science projects use BOINC. They investigate diseases, study climate change, discover pulsars, and do many other types of scientific research.
+  The BOINC and Science United projects are located at the University of California, Berkeley and are supported by the National Science Foundation.
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: devmode # use 'strict' once you have the right plugs and slots
+
+apps:
+  boinc:
+    command: usr/bin/boinc --dir $HOME
+    plugs:
+    - network
+    - network-bind
+
+  client:
+    command: usr/bin/boinc --dir $HOME
+    plugs:
+    - network
+    - network-bind
+
+  manager:
+    command: usr/bin/boincmgr --datadir $HOME
+    plugs:
+    - network
+    - network-bind
+    extensions:
+    - gnome    
+    slots:
+    - dbus-daemon
+    common-id: boinc.manager
+    environment:
+      XAUTHORITY: $SNAP_REAL_HOME/.Xauthority
+
+slots:
+  dbus-daemon:
+    interface: dbus
+    bus: session
+    name: boinc.manager
+
+parts:
+  boinc:
+    build-packages:
+    - libftgl-dev
+    - freeglut3-dev
+    - libnotify-dev
+    - libxmu-dev
+    - libxi-dev
+    - libfcgi-dev 
+    - libxss-dev 
+    - libxcb-util0-dev
+    - libxxf86vm-dev
+    - libxrender-dev
+    - libxrandr-dev
+    - libdbus-1-dev
+    - libxtst-dev
+    # vcpkg
```

Comment:
Do we really need it here?

Reply:
Yes

Reply:
Then why it's commented out?

Reply:
It a comment to keep things organize.

Reply:
I change the comments to be more cleat that it is a comments.